packages/boot/src/__tests__/acceptance/crud-rest-builder.acceptance.ts

```diff
@@ -0,0 +1,168 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {juggler, RepositoryMixin} from '@loopback/repository';
+import {RestApplication} from '@loopback/rest';
+import {
+  CrudRestComponent,
+  defineCrudRepositoryClass,
+} from '@loopback/rest-crud';
+import {expect, givenHttpServerConfig, TestSandbox} from '@loopback/testlab';
+import {resolve} from 'path';
+import {BootMixin, ModelApiBooter} from '../..';
+import {Product} from '../fixtures/product.model';
+
+describe('CRUD rest builder acceptance tests', () => {
+  let app: BooterApp;
+  const SANDBOX_PATH = resolve(__dirname, '../../.sandbox');
+  const sandbox = new TestSandbox(SANDBOX_PATH);
+
+  beforeEach('reset sandbox', () => sandbox.reset());
+  beforeEach(givenAppWithDataSource);
+
+  afterEach(stopApp);
+
+  it('binds the controller and repository to the application', async () => {
+    await sandbox.copyFile(
+      resolve(__dirname, '../fixtures/product.model.js'),
+      'models/product.model.js',
+    );
+
+    await sandbox.writeTextFile(
+      'model-endpoints/product.rest-config.js',
+      `
+const {Product} = require('../models/product.model');
+module.exports = {
```

Comment:
we don't need to specify the type like documented [in the doc](https://github.com/strongloop/loopback-next/blob/bcc3000d55f457c15463cf43ee18091dcc0dd09f/packages/rest-crud/README.md#basic-use)?
```ts
module.exports = <ModelCrudRestApiConfig>{
  model: Product,
  pattern: 'CrudRest',
  dataSource: 'db',
  basePath: '/products',
};
```

Reply:
Since we're using a js file (model-endpoints/product.rest-config.js) instead of a ts file, the type can't be specified like it is in the docs (or else it'll have compilation errors).